ubuntu/README.md

```diff
@@ -0,0 +1,91 @@
+# The Flutter Ubuntu Desktop Embedder
+
+This framework provides the basic functionality for embedding a Flutter app on
+the Ubuntu platform. This currently includes support for:
+
+*  Drawing a Flutter view.
+*  Mouse event handling.
+
+# How to Use This Code
+
+Note that the example build here is `host_debug_unopt` (you'll find more info on
+what that means in this section). This can be changed to whatever version of
+the Flutter Engine you end up building.
+
+First you will need to install the relevant dependencies.
+
+## Dependencies
+
+```
+$ sudo apt-get install libglfw3-dev
+```
+
+Next you will need to follow the steps outlined on the flutter engine's
+[contributing](https://github.com/flutter/engine/blob/master/CONTRIBUTING.md)
+document so that you can build a copy of `libflutter_engine.so`.
+
+Then copy it in the `ubuntu/` directory.
+
+```
+$ cp <path_to_flutter_engine>/src/out/host_debug_unopt/libflutter_engine.so \
+       <path_to_flutter_desktop_embedding_repo>/ubuntu/
+```
+
+Then copy `embedder.h` into the same directory (the link can be found on [this
+page](https://github.com/flutter/engine/wiki/Custom-Flutter-Engine-Embedders),
+under [this
+link](https://github.com/flutter/engine/blob/4733e3373789894aa4f593137c6d440891d492a2/shell/platform/embedder/embedder.h)).
+
+```
+$ cd <path_to_flutter_desktop_embedding_repo>
+$ wget
+https://raw.githubusercontent.com/flutter/engine/4733e3373789894aa4f593137c6d440891d492a2/shell/platform/embedder/embedder.h
+```
+
+After that's all done you should be able to run `make` and have access to a
+`flutter_embedder` binary.
+
+```
+$ make
+```
+
+After that you'll probably want to run an example Flutter app if you don't
+already have one.
```

Comment:
I'd like to follow the pattern of having the example be something that's hard-coded to run /example_flutter, rather than being a generic launcher.

Reply:
Done.